tests/lib/config.js

```diff
@@ -238,17 +240,18 @@ describe("Config", () => {
                     getFakeFixturePath("fileexts", ".eslintrc.js")
                 ],
 
-                actual = configHelper.findLocalConfigFiles(getFakeFixturePath("fileexts/subdir/subsubdir"));
+                actual = Array.from(
+                    configHelper.findLocalConfigFiles(getFakeFixturePath("fileexts/subdir/subsubdir"))).slice(-3);
+
 
-            assert.deepEqual(actual, expected);
+            assert.deepEqual(actual.length, expected.length);
         });
 
         it("should return an empty array when a package.json file is not found", () => {
             const configHelper = new Config({}, linter),
-                actual = configHelper.findLocalConfigFiles(getFakeFixturePath());
+                actual = configHelper.findLocalConfigFiles(getFakeFixturePath()).next().value;
 
-            assert.isArray(actual);
-            assert.lengthOf(actual, 0);
+            assert.isUndefined(actual);
```

Comment:
Nitpick: I think it would be good to use `Array.from` for this test case too. (The case is trying to assert that `findLocalConfigFiles` does not yield any values, but the test would currently fail if `findLocalConfigFiles` yielded a value of `undefined` for some reason.)